pkgs/development/python-modules/django-csp-nonce/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchPypi, buildPythonPackage, django }:
+
+buildPythonPackage rec {
+  pname = "django_csp";
```

```suggestion
  pname = "django-csp";
```

We'd like to normalize `pname` in this manner.

You'll need to change the `pname` in `fetchPypi` or else the fetch will break.

---

sure!